### PR TITLE
[stable/influxdb] Add startup probe to stable/influxdb deployment

### LINKED ISSUE #18167: [stable/influxdb] Define startup probe in deployment
**Describe the bug**
Please define a startup probe in the deployment file so we can easily control pod startup timeouts. Large Influx DBs take a while to start -- right now my influx DB pod consumes 80GB of memory and takes about 48 minutes to cold-start.

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes

**Version of Helm and Kubernetes**:
```
jharris@jharris-XPS-15-9570:~/Repository/helm.influxdb$ helm version
Client: &version.Version{SemVer:"v2.14.3", GitCommit:"0e7f3b6637f7af8fcfddb3d2941fcc7cbebb0085", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.14.3", GitCommit:"0e7f3b6637f7af8fcfddb3d2941fcc7cbebb0085", GitTreeState:"clean"}
```

```
jharris@jharris-XPS-15-9570:~/Repository/helm.influxdb$ kubectl version
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.3", GitCommit:"2d3c76f9091b6bec110a5e63777c332469e0cba2", GitTreeState:"clean", BuildDate:"2019-08-19T11:13:54Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.3", GitCommit:"2d3c76f9091b6bec110a5e63777c332469e0cba2", GitTreeState:"clean", BuildDate:"2019-08-19T11:05:50Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Which chart**:
stable/influxdb

**What happened**:
pod is prematurely restarted by kubernetes when upgrading/installing influxdb chart with a database >= 80GB

**What you expected to happen**:
helm chart to be configured with deployment startup probe support for large databases

**How to reproduce it** (as minimally and precisely as possible):
install the chart with a large influxdb database

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by reloaded**
#### What this PR does / why we need it:
Add kubernetes deployment startup probe to stable/influxdb deployment.

#### Which issue this PR fixes
  - fixes #18167 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by reloaded** [comment]
Please review @jackzampolin  @Aisuko

File: stable/influxdb/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: influxdb
-version: 1.4.1
+version: 1.4.2
```
Comment (Aisuko) [inline stable/influxdb/Chart.yaml]: Please update the minor version, `1.5.0` could be good.

**Review by Aisuko — CHANGES_REQUESTED** [review]
Thanks for your PR @reloaded. Please signed off first.

**Comment by reloaded** [comment]
@Aisuko  This PR is ready for you to review again.

**Review by Aisuko — APPROVED** [review]
/LGTM

**Comment by Aisuko** [comment]
Thanks a lot.

**Pull request merged**